The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/15863121808/job/44724661114) [(plain)](https://github.com/rust-lang/rust/commit/92b51e920033508101ed2b0b09a090fc2ac5fdee/checks/44724661114/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
[TIMING] core::build_steps::tool::Tidy { compiler: Compiler { stage: 0, host: x86_64-unknown-linux-gnu, forced_compiler: false }, target: x86_64-unknown-linux-gnu } -- 0.000
fmt check
fmt: checked 6122 files
tidy check
tidy error: Dependency for . not explicitly permitted: registry+https://github.com/rust-lang/crates.io-index#semver@1.0.26
Go to `src/tools/tidy/src/deps.rs:242` for the list.
Checking tidy rustdoc_json...
Running eslint on rustdoc JS files
`rustdoc-json-types` was not modified.
No error code explanation was removed!
tidy: Skipping binary file check, read-only filesystem
---
All checks passed!
checking python file formatting
28 files already formatted
checking C++ file formatting
some tidy checks failed
Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:01:17
  local time: Tue Jun 24 22:55:05 UTC 2025
  network time: Tue, 24 Jun 2025 22:55:05 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/src/tools/tidy/src/deps.rs
+++ b/src/tools/tidy/src/deps.rs
@@ -374,6 +374,7 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "scoped-tls",
     "scopeguard",
     "self_cell",
+    "semver",
     "serde",
     "serde_derive",
     "serde_json",

--- a/src/tools/wasm-component-ld/Cargo.toml
+++ b/src/tools/wasm-component-ld/Cargo.toml
@@ -10,4 +10,4 @@ name = "wasm-component-ld"
 path = "src/main.rs"
 
 [dependencies]
-wasm-component-ld = "0.5.13"
+wasm-component-ld = "0.5.14"

